src/telliot_feed_examples/cli.py

```diff
@@ -143,11 +149,20 @@ def cli(
 
 # Report subcommand options
 @cli.command()
+@click.option(
+    "--query-tag",
+    "-qt",
+    "query_tag",
+    help="select datafeed using query tag",
+    required=False,
+    nargs=1,
+    type=click.Choice([q.tag for q in query_catalog.find()]),
+)
 @click.option(
     "--legacy-id",
     "-lid",
     "legacy_id",
-    help="report to a legacy ID",
+    help="select datafeed using legacy ID",
```

Comment:
Not sure there's any benefit to maintaining the legacy ID option.

Reply:
I thought to keep it just because some people are used to that naming, but probably not a big deal, so I'll take it out.